### PR TITLE
docs(instance): clarify the ipmob documentation

### PR DESCRIPTION
**Pull request by ziirish**
### Your checklist for this pull request

- [ ] Check that the commit messages match our requested structure.
- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description
Please describe your pull request.

### CODE REVIEW

File: compute/instances/api-cli/using-ip-mobility.mdx [hunk 1]
```diff
@@ -44,7 +45,7 @@ Another feature of IP mobility is the ability to assign multiple flexible IP add
 In this section, we will guide you through the process of creating a new IP mobility-enabled Instance. We will also provide instructions on migrating your existing legacy Instances.
 
  * The following examples rely on [httpie](https://httpie.io/cli), you can easily adapt them with curl if you prefer.
- * The `API_URL` variable contains the following value: https://api.scaleway.com/instance/v1/zones/pl-waw-2
+ * The `API_URL` variable contains the following value: https://api.scaleway.com/instance/v1/zones/<zone>
```
Comment (bene2k1) [inline compute/instances/api-cli/using-ip-mobility.mdx:48]:
```suggestion
 * The `API_URL` variable contains the following value: `https://api.scaleway.com/instance/v1/zones/<zone>`
```

Comment (ziirish) [reply compute/instances/api-cli/using-ip-mobility.mdx]: Thanks

File: compute/instances/api-cli/using-ip-mobility.mdx [hunk 2]
```diff
@@ -28,11 +28,12 @@ This change can occur when you stop and restart your Instance. This private netw
...
-Another feature of IP mobility is the ability to assign multiple flexible IP addresses to your Instance, allowing you to choose between IPv4 and IPv6 protocols.
+Another feature of IP mobility is the ability to assign multiple flexible IP addresses to your Instance, allowing you to use both IPv4 and IPv6 protocols.
 
 <Message type="note">
-   * Currently, IP mobility is available in the `pl-waw-2` Availability Zone.
-   * [Dynamic IPs](/compute/instances/concepts/#dynamic-ip) continue to be supported after migration to IP mobility.
+   * Currently, IP mobility is available in the `pl-waw-2`, `fr-par-3` and `nl-ams-2` Availability Zones.
+   * [Dynamic IPs](/compute/instances/concepts/#dynamic-ip) continue to be supported after the migration to IP mobility.
+   * The field `enable_ipv6` is being deprecated and has no meaning for IP Mobility enabled instances.
```
Comment (RoRoJ) [inline compute/instances/api-cli/using-ip-mobility.mdx:36]:
```suggestion
   * The field `enable_ipv6` is being deprecated and has no meaning for IP mobility enabled instances.
```

**Review by RoRoJ — APPROVED** [review]

**Review by farhakennar — APPROVED** [review]

File: compute/instances/api-cli/using-ip-mobility.mdx (hunk 2 above)
Comment (ziirish) [reply compute/instances/api-cli/using-ip-mobility.mdx]: I have updated this part. Thanks!

**Pull request merged**